homeassistant/components/thermostat/__init__.py

```diff
@@ -301,22 +306,32 @@ def turn_fan_off(self):
     @property
     def min_temp(self):
         """Return the minimum temperature."""
-        return convert(7, TEMP_CELSIUS, self.unit_of_measurement)
+        unit = self.unit_of_measurement
+        if not unit:
```

Comment:
Oh boy - this changes a lot of existing behavior. I totally agree with it though.